src/components/live-announcer/live-announcer.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  Component,
+  ElementRef,
+  AfterContentInit
+} from 'angular2/core';
+
+import {DOM} from '../../core/platform/dom/dom_adapter';
+
+@Component({
+  selector: 'md-live-announcer',
+  templateUrl: './components/live-announcer/live-announcer.html',
+  styleUrls: ['./components/live-announcer/live-announcer.css'],
+})
+export class MdLiveAnnouncer implements AfterContentInit {
+
+  private announcerEl: HTMLElement;
+
+  constructor(private elementRef: ElementRef) { }
+
+  ngAfterContentInit() {
+    this.announcerEl = DOM.querySelector(this.elementRef.nativeElement, '.md-live-announcer');
```

Comment:
Why did you change this away from using the `Renderer` ? If the binding doesn't work, using the `Renderer` is preferable.